docs/api-testing/mark3/quick-start/integrate-with-your-cicd.md

```diff
@@ -0,0 +1,135 @@
+
+The webhooks are organized in the following manner:
+* `username`: your Sauce Labs user name
+* `access_key`: your Sauce Labs access key
+* `hookId`: generated by API Fortress Mark3 cloud
+
+```http request
+https://{username}:{access_key}@domain/api-testing/rest/v4/{hookId}
+```
+
+To run the CLI tool:
+
+```bash
+docker run quay.io/saucelabs/apifctl
+```
+
+### Generating a Webhook
+
+To create a webhook:
+
+1. Navigate to the project window and select the _WebHooks_ tab:
+   <img src={useBaseUrl('img/api-fortress/2021/04/webHooksSection.png')} alt="webhook screenshot"/>
+
+1. Select **Create Hook**
+   <img src={useBaseUrl('img/api-fortress/2021/04/createHook.png')} alt="Create New WebHook"/>
+
+1. Enter the field details and select **Save**
+   <img src={useBaseUrl('img/api-fortress/2021/04/sampleHook.png')} alt="sample webhook details"/>
+
+1. A generated _Hook URL_ appears. Copy it to your clipboard and use it either locally or as part of CI build (the URL also appears in the [user settings](https://app.saucelabs.com/user-settings) section)
+   <img src={useBaseUrl('img/api-fortress/2021/04/hookURL.png')} alt="sample Hook URL"/>
+    
+## Commands
+
+### `run`
+
+__Description__: Runs a single test. The `{hookId}` and `testId` are both required.
+
+__Example__:
+```bash
+docker run quay.io/saucelabs/apifctl run -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5 -i 555161be0ad1b20c6e2dd020
+```
+---
+
+### `run-all`
+
+__Description__: Runs all the complete tests in the project referenced by the hook.
+
+__Example__:
+```bash
+docker run quay.io/saucelabs/apifctl run-all -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5
+```
+---
+
+### `run-tag`
+
+__Description__: Runs all the tests tagged with a specific tag.
+
+__Example__:
+```bash
+docker run quay.io/saucelabs/apifctl run-tag -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5 -tag production
+```
+---
+
+### `exec`
+
+__Description__: Sends a test residing in the local file system (directory with `unit.xml` and `input.xml`) to the cloud for execution.
+
+__Example__:
+```bash
+docker run -v $(pwd)/tests:/tests quay.io/saucelabs/apifctl exec -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5 -p /tests/test_abc
+```
+---
+
+### `upload`
+
+__Description__: Uploads a test residing in the local file system  (directory with `unit.xml` and `input.xml`) to the cloud for storage.
+
+__Example__:
+```bash
+docker run -v $(pwd)/tests:/tests quay.io/saucelabs/apifctl upload -H https://foobar:ae5499d58ab1@api.us-west-1.saucelabs.com/api-testing/rest/v4/4be5-45db-a9aa-4c5d9027e6f5 -p /tests/test_abc
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Username And Password In Uri' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

 <!-- d53e3e54-dcf9-4822-a167-61a8cefc292c -->